src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -156,7 +156,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.18.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.0.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.1.3" />
```

Comment:
Hi, you need to make sure the workers match the v4.1.3 tag in the host repo. It appears java is the only one that doesn't match

https://github.com/Azure/azure-functions-host/blob/v4.1.3/src/WebJobs.Script/WebJobs.Script.csproj#L52

Reply:
@VpOfEngineering 
Can you please take a look at this

Reply:
Thank you @ejizba, @shreyas-gopalakrishna  please can you confirm?

Reply:
@v-vishalr  For v4.1.3 we want the 2.0.0 java worker

Reply:
@VpOfEngineering updated java worker to 2.0.0

Reply:
@VpOfEngineering  updated